src/cli.py

```diff
@@ -189,7 +193,7 @@ def reset_all(opts, conf):
     if all_nodes is None:
         logging.error("Could not get nodes to reset")
         all_nodes = []
-    nodes_to_delete = [x for x in all_nodes if x != zk.MEMBERS_PATH]
+    nodes_to_delete = [x for x in all_nodes if x not in (zk.MEMBERS_PATH, zk.MAINTENANCE_PATH)] + [zk.MAINTENANCE_PATH]
```

Comment:
Do you mean here, that `zk.MAINTENANCE_PATH` node should be deleted last because it is checked first in every iteration, and it is a barrier against races with working pgconsul?
It is true when maintenance is enabled, but not true if not; there can be other races. (UPD.: now I see this is the case; it would be more clear to add `zk.MAINTENANCE_PATH` in the end, after main deletion cycle).
Generally, we need to establish an order of importance of this keys, and use it when cleaning (my #50 is actually about the same thing).

But that's for future discussion, not a topic for this PR.